test/performance/Hmac/HmacBench.php

```diff
@@ -11,7 +11,7 @@
 /** @Groups({"Hmac"}) */
 abstract class HmacBench extends SignerBench
 {
-    private const ENCODED_KEY = 'hJtXIZ2uSN5kbQfbtTNWbpdmhkV8FJG+Onbc6mxCcYg';
+    private const ENCODED_KEY = 'hJtXIZ2uSN5kbQfbtTNWbpdmhkV8FJG+Onbc6mxCcYg=';
```

Comment:
Previously `\Lcobucci\JWT\Signer\Key\InMemory::base64Encoded` accepted both paddad and non-padded base64 strings due to `base64_decode` looseness.

For the purpose of `InMemory` class, I chose `SODIUM_BASE64_VARIANT_ORIGINAL` and I chose not to accept (alias fallback) non-padded keys.

I consider this sort of a bug, so no BC break for me here.